changelog/fix-8706-tpv-view-report-link

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fix
+Comment: Fix the `View report` link on Total payment volume tile on the Payment activity widget. Changes behind feature flag, and part of larger change that adds the wPayment activity widget.
```

Comment:
```suggestion
Comment: Fix the `View report` link on Total payment volume tile on the Payment activity widget. Changes behind feature flag, and part of larger change that adds the Payment activity widget.
```

Reply:
Sorry about that typo. Thanks for catching it! - 7930735